implementations/rust/ockam/ockam_command/src/trust_context/delete.rs

```diff
@@ -33,31 +31,19 @@ impl DeleteCommand {
 fn run_impl(opts: CommandGlobalOpts, cmd: DeleteCommand) -> miette::Result<()> {
     let DeleteCommand { name } = cmd;
     let state = opts.state.trust_contexts;
-    match state.get(&name) {
-        // If it exists, proceed
-        Ok(_) => {
-            if let ConfirmResult::No = opts.terminal.confirm(&fmt_warn!(
-                "This will delete the trust context with name '{name}'. Do you want to continue?"
-            ))? {
-                // If the user has not confirmed, exit
-                return Ok(());
-            }
-
-            state.delete(&name)?;
-
-            opts.terminal
-                .stdout()
-                .plain(fmt_ok!("Trust context with name '{name}' has been deleted"))
-                .machine(&name)
-                .json(serde_json::json!({ "trust-context": { "name": &name } }))
-                .write_line()?;
-
-            Ok(())
-        }
-        // Else, return the appropriate error
-        Err(err) => match err {
-            CliStateError::NotFound => Err(miette!("Trust context '{name}' not found")),
-            _ => Err(err.into()),
-        },
+    state.get(&name)?;
+    if let ConfirmResult::No = opts.terminal.confirm(&fmt_warn!(
+        "This will delete the trust context with name '{name}'. Do you want to continue?"
```

Comment:
Perhaps rephrase with: "This will delete the trust context named '{name}'. Do you wish to proceed?"